file_store/src/heartbeat.rs

```diff
@@ -76,6 +79,51 @@ impl MsgTimestamp<Result<DateTime<Utc>>> for CellHeartbeatIngestReportV1 {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ValidatedHeartbeat {
```

Comment:
NOTE, there is an existing ValidatedHeartbeat struct in mobile verifier...which does already include all the data you require here.  Have you considered if these can be harmonized ?

Reply:
missed that those exist, will look at pulling those into file_store